build failed. not obvious since the last commit uses [ci skip]
```
========================================
FAIL: test_checkpoint_with_multiple_topics (corehq.apps.change_feed.tests.test_kafka_change_feed.KafkaCheckpointTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/decorators.py", line 24, in decorated
    return fn(*args, **kwds)
  File "/mnt/commcare-hq-ro/corehq/apps/change_feed/tests/test_kafka_change_feed.py", line 97, in test_checkpoint_with_multiple_topics
    self.assertEqual(feed.get_current_checkpoint_offsets(), pillow.get_last_checkpoint_sequence())
AssertionError: {'case': 5, 'form': 2} != {u'case': 4, u'form': 2}
- {'case': 5, 'form': 2}
?          ^
+ {u'case': 4, u'form': 2}
?  +        ^  +
----------------------------------------------------------------------
```

--- a/corehq/apps/change_feed/consumer/feed.py
+++ b/corehq/apps/change_feed/consumer/feed.py
@@ -93,8 +93,8 @@ class KafkaChangeFeed(ChangeFeed):
 
         try:
             for message in consumer:
-                yield change_from_kafka_message(message)
                 self._processed_topic_offsets[message.topic] = message.offset
+                yield change_from_kafka_message(message)
         except ConsumerTimeout:
             assert not forever, 'Kafka pillow should not timeout when waiting forever!'
             # no need to do anything since this is just telling us we've reached the end of the feed

